RoktDemo/Model/DemoLibraryModel.swift

```diff
@@ -18,7 +18,7 @@ struct DemoLibraryModel: Codable {
     let demoDescription: String
     let defaultPlacementsExamples: DefaultPlacementExamplesModel
     let customConfigurationPage: CustomConfigurationPageModel
-    let preDefinedScreen1: PreDefinedScreen1Model
-    let preDefinedScreen2: PreDefinedScreen2Model
-    let preDefinedScreen3: PreDefinedScreen3Model
+//    let preDefinedScreen1: PreDefinedScreen1Model
```

Comment:
@DJingSeo I am going to uncomment these lines in the next PR after the first release.
The only reason I commented these lines is to avoid API versioning if we decided to change the structure of predefined screens.